README.md

```diff
@@ -2,8 +2,8 @@
 
 To start the slide show:
 
-- `pnpm install`
-- `pnpm run dev`
+- `pixi run pnpm install`
+- `pixi run pnpm run dev`
```

Comment:
How do you feel about encapsulating those into pixi tasks?

Reply:
Done in https://github.com/Hofer-Julian/slidev-presentations/pull/1/commits/a145e209491daf2ae3ee3ee46edda9aa7e91511c .